includes/pages/user_shifts.php

```diff
@@ -197,8 +198,11 @@ function load_types()
                 ON (
                     `user_angel_type`.`angel_type_id`=`angel_types`.`id`
                     AND `user_angel_type`.`user_id`=?
-                )
-            ORDER BY `angel_types`.`name`
+                )'
+            . ($isShico ? '' :
+            'WHERE angel_types.hide_on_shift_view = 0
+            OR user_angel_type.user_id IS NOT NULL ') .
```

Comment:
this indentation does not feel right, imho some spaces should be added here?

Reply:
please be more precise
done